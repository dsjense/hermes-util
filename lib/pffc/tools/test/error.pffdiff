# $Id$
# 
# Copyright (2008) Sandia Corporation. Under the terms of
# Contract DE-AC04-94AL85000 with Sandia Corporation, the U.S.
# Government retains certain rights in this software.
# 
# Hermes is free software: you can redistribute it and/or modify
# it under the terms of the GNU Lesser General Public License as
# published by the Free Software Foundation, either version 3 of
# the License, or (at your option) any later version.
# 
# Hermes is distributed in the hope that it will be useful, but
# WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU Lesser General Public License for more details.
# 
# You should have received a copy of the GNU Lesser General
# Public License along with Hermes.  If not, see
# <http://www.gnu.org/licenses/>.
# 
base, fi=tqhis.base_pff
mean,ti=ekill ds=3 /lim 
test, xxx=tqhis.pff
test, fi=tqhis.pff
mean,ti=ekill ds=3 /lim 
mean,ti=ekill ds=4
qqq, fi=2, xx=yy
rms, lim=5 abc=20
max, ti=vin lim=1e-3
