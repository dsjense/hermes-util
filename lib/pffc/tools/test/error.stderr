Parsing Error in error.pffdiff[23]: Unrecognized Keyword
   xxx
Parsing Error in error.pffdiff[27]: Unknown command
   qqq
Parsing Error in error.pffdiff[28]: Unknown keyword: 
   abc
